src/components/PageSwitcher/components/EEGEduEvoked/sketchEvoked.js

```diff
@@ -0,0 +1,64 @@
+export default function sketchEvoked (p) {
+
+ let x = 0;
+ let thisRand = 0.5; //for random choice of target type
+ let targProp = 0.25;
+ let isTarget = false;
+ 
+
+ p.setup = function () {
+    p.createCanvas(300, 300);
+    p.frameRate(30);
```

Comment:
added this to this and the ssvep animations to aim for lowest common denominator of hardware people may have, and set this common amongst them.